exposure-notification/README.md

```diff
@@ -48,6 +48,10 @@ When returning data from cache, those loggings are left out, so this should corr
 
 `MaintenanceService` logs numeric information about background tasks that clean up old data and pre-cache batches.
 
+In addition, for class level logging, Mapped Diagnostic Context (MDC) is used to add pseudorandom correlation identifier (correlationId)
+to every log message. CorrelationId binds all individual log messages of request flow together. For compatibility reasons,
+correlationId is called errorId in http-responses.
```

Comment:
The error id is mentioned above under ApiErrorHandler as well. That mention is now slightly deprecated - these should be combined. Same thing in the other README

Reply:
ok, sanitized documentation a little bit.